actions/post_actions.test.js

```diff
@@ -92,6 +96,7 @@ describe('Actions.Posts', () => {
                 channels: {
                     current_channel_id: {team_a: 'team_a', id: 'current_channel_id'},
                 },
+                manuallyUnread: {},
```

Comment:
Adding this causes an async action to no longer error out which means some extra actions are recorded by the mock store below.